python/CMakeLists.txt

```diff
@@ -534,6 +539,20 @@ if(PYARROW_BUILD_FLIGHT)
   set(CYTHON_EXTENSIONS ${CYTHON_EXTENSIONS} _flight)
 endif()
 
+# Engine
+
+if(PYARROW_BUILD_ENGINE)
+find_package(ArrowEngine REQUIRED)
+  if(PYARROW_BUNDLE_ARROW_CPP)
+    message("ARROW_ENGINE_SHARED_LIB")
+    message(""${ARROW_ENGINE_SHARED_LIB})
```

Comment:
Are these log lines necessary?

Reply:
Oh this was a typo. I was checking the cmake build.